### ISSUE TITLE
getDeferredApplink is not working

### ISSUE THREAD

**Issue by evyatar** [jeduan/cordova-plugin-facebook4#411]
Calling to `facebookConnectPlugin.getDeferredApplink(function(url){ /*...*/ }, function(err){ /*...*/ });` always return an error "Invalid action" (to the error callback). tested only on android.

bty, the `exports.getDeferredApplink = function (s, f) { /*....*/ }` appears twice on the facebook-native.js file.

**Comment by mpk2** [comment]
This doesn't occur on iOS but can confirm this issue on Android. Running Nexus 5X on Android 7.1.1

**Comment by mpk2** [comment]
It looks like the android plugin is just totally missing the functionality.

In `ConnectPlugin.java`:

```
@Override
    public boolean execute(String action, JSONArray args, final CallbackContext callbackContext) throws JSONException {
        if (action.equals("login")) {
            ...
        } else if (action.equals("logout")) {
            ...
        } else if (action.equals("getLoginStatus")) {
            ...
        } else if (action.equals("getAccessToken")) {
            ...
        } else if (action.equals("logEvent")) {
            ...
        } else if (action.equals("logPurchase")) {
            ...
        } else if (action.equals("showDialog")) {
            ...
        } else if (action.equals("graphApi")) {
            ...
        } else if (action.equals("appInvite")) {
           ...
        } else if (action.equals("activateApp")) {
            ...
        }
        return false;
    }
```

even though `facebook-native.js` declares 
```
exports.getDeferredApplink = function (s, f) {
  exec(s, f, 'FacebookConnectPlugin', 'getDeferredApplink', [])
}
```

**Comment by mpk2** [comment]
Looks like there is more info at #217, and it got merged in #362 , which seems to be plugin version 1.7.4. The above seems to be my plugin at ~1.7.1 instead

**Comment by evyatar** [comment]
As @mpk2 mentioned. Manually updating the ConnectPlugin.java file (located in platforms\android\src\org\apache\cordova\facebook) solved the issue!
